dashboard.go

```diff
@@ -31,6 +35,24 @@ type Dashboard struct {
 	Overwrite bool                   `json:overwrite`
 }
 
+// Dashboards represent json returned by search API
+type Dashboards struct {
```

Comment:
If this is specific to the search endpoint can we name it something more along the lines of `DashboardSearchResponse`?